test/e2e/framework/framework.go

```diff
@@ -43,6 +43,7 @@ const (
 	groupName       = "interconnectedcloud.github.io"
 	apiVersion      = "v1alpha1"
 	opImage         = "quay.io/interconnectedcloud/qdr-operator"
```

Comment:
Suggest removing opImage and qdrImage from this struct and just refer to TestContext.* where appropriate

Reply:
These two constants: opImage and qdrImage are being used as the "default" values for the respective command line arguments. Do you think we should have them defined directly the in the RegisterFlags method and remove them from this list of constants?